db/migrate/20160425181824_change_tasks_to_include_completion_indicator.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeTasksToIncludeCompletionIndicator < ActiveRecord::Migration
+  def change
+  	add_column :tasks, :completed_bool, :boolean
+  end
+end
```

Comment:
I like how you indicated in the name that the completed column is a boolean.  This is helpful since completed could also be another datatype